### DISCUSSION TITLE
CI: Travis ...

### DISCUSSION THREAD

**Discussion by micaelmalta** [dogecoin/dogecoin#2302] [Q&A]
Hello everyone!

As the community is growing and we have a lot of developments going on, we are actually blocked by Travis Plan.
We should maybe migrate cirrus (same as Bitcoin) which is free for Open Source projects. 

Also, we can see that on Travis we build 2 tasks while bitcoin build way more: is it a restriction because of Travis?

What do you think?

**Comment by rnicoll** [comment]
I suspect Cirrus is probably the next place to look, I like the idea of GitHub actions but @patricklodder 's mentioned he's seen infrastructure failures on actions trigger false-positives.

**Reply by patricklodder** [reply]
I'm okay with trying GH actions. Just need to sometimes run local tests just in case...

**Comment by patricklodder** [comment]
I'm currently setting up GH Actions for 1.14.4-dev, and @rnicoll is looking at Cirrus for 1.21-dev

This gives us a way to do a proper comparison.

**Reply by micaelmalta** [reply]
@rnicoll I have a work in progress for 1.21-dev and cirrus 
https://github.com/mmicael1/dogecoin/pull/8
https://cirrus-ci.com/build/6339211902058496

**Reply by micaelmalta** [reply]
Good point for Cirrus: you can have up to 16 CPU simultaneously :)

**Comment by amahpour** [comment]
Has anyone considered Gitlab? They also have free pipelines for open-source projects.
